examples/gno.land/p/demo/int256/arithmetic_test.gno

```diff
@@ -192,7 +195,9 @@ func TestSub(t *testing.T) {
 		{"-1", "-1", "0"},
 		{"-115792089237316195423570985008687907853269984665640564039457584007913129639935", "-115792089237316195423570985008687907853269984665640564039457584007913129639935", "0"},
 		{"-115792089237316195423570985008687907853269984665640564039457584007913129639935", "0", "-115792089237316195423570985008687907853269984665640564039457584007913129639935"},
```

Comment:
I get why this test works, but it's weird. Can we change the test so that instead of parsing also the `want`, we simply compare the string? So we verify if it's the "canonical" representation?

Reply:
This goes for other tests as well